### ISSUE TITLE
Dexie performance degrades really badly when the tab running Dexie.js is set to inactive/blurred

### ISSUE THREAD

**Issue by lpetrov** [dfahlander/Dexie.js#442]
After playing with Dexie.js for a while, I'd found out that when the tab running Dexie.js is not active, Dexie.js operations start to get REALLY, REALLY slow.

I'd prepared a example test case scenario, by using the "hello world" fiddle provided on dexie.org, here it is:
https://jsfiddle.net/w2w01h5c/2/

Just change the 'thisIsAnInactiveTabTest' flag to see the different time take for db setup, db add + db read when the tab is active/not.

**Comment by nponiros** [comment]
From what I know this is a browser issue. This also happens with timeouts from what I know and any other operation. I guess it is an optimization so that inactive tabs don't use too many resources.

**Comment by lpetrov** [comment]
Looks like this is in fact an issue in Chrome:
https://bugs.chromium.org/p/chromium/issues/detail?id=675372

**Comment by bman654** [comment]
I ran into this issue today.  Updating Chrome to 56.0.2924.87 resolved the problem.

**Comment by lpetrov** [comment]
Yup, in the link to bugs.chromium.org, you can see that they had recently fixed it.
